test/k8sT/Chaos.go

```diff
@@ -75,17 +75,28 @@ var _ = Describe("K8sChaosTest", func() {
 		Expect(err).To(BeNil(), "Cannot get daemonset pods IPS")
 		Expect(len(pods)).To(BeNumerically(">", 0), "No pods available to test connectivity")
 
+		By("\tWaiting for kube-dns entry for service testds-service")
```

Comment:
Due the place of this By, I don't think that the `\t` make sense here.

Reply:
I'll just remove the use of `\t` from `PingService`.

Reply:
Removed use of tabs.